react-native-branch-segment.podspec

```diff
@@ -0,0 +1,29 @@
+require 'json'
+
+# Podspec specifically for use in hybrid apps with the Branch Segment integration.
+# Not for use in pure RN apps.
+
+# expect package.json in current dir
+package_json_filename = File.expand_path("./package.json", __dir__)
+
+# load the spec from package.json
+spec = JSON.load(File.read(package_json_filename))
+
+Pod::Spec.new do |s|
+  s.name             = "#{spec['name']}-segment"
+  s.version          = spec['version']
+  s.summary          = spec['description']
+  s.requires_arc = true
+  s.authors      = {
+                     'rt2zz' => 'zack@root-two.com',
+                     'Jimmy Dee' => 'jgvdthree@gmail.com'
+                   }
+  s.license      = spec['license']
+  s.homepage     = spec['homepage']
+  s.platform     = :ios, "7.0"
+  s.source       = { spec['repository']['type'].to_sym => spec['repository']['url'].sub(/^[a-z]+\+/, '') }
+  s.source_files = [ "ios/*.h", "ios/*.m"]
+  s.header_dir   = "react-native-branch"
```

Comment:
This ensures that imports don't have to change, if you're importing react-native-branch in your native code. This isn't actually necessary unless you're using `branch.subscribe` in your RN (JS) code. If you're routing links in native code, which seems the most likely thing, you can just use the Branch native iOS SDK directly. This is how webview_example_native_ios is set up. The testbed_native_ios app uses the RNBranch native SDK as an example and verifies that this change builds correctly.